index.rst

```diff
@@ -1,59 +1,367 @@
+Discrete SkyMaps, which typically cover only a small part of the sky and are *conceptually* a bit more per-user, maybe less rare, but our data model currently does not treat these any differently, and until we can identify the patterns and use cases for creating new SkyMaps (even discrete ones), we propose that any new SkyMap registration in a shared repository be preceded by an RFC.
+
+.. _collections-shared-official-processing-outputs:
+
+Shared/official processing outputs
+----------------------------------
+
+Processing runs overseen by production operators should produce output collections of the form ``<instrument>/runs/<target>/<release>/<ticket>``, or ``runs/<target>/<release>/<ticket>`` in the (rare) case of processing that includes science data from multiple instruments and none of them can be considered the "primary" instrument.
+``<target>`` is a human-meaningful name for the set of data IDs being processed, and ``<release>`` is some kind of DM software release version, so examples of complete processing-output collection names might include ``HSC/runs/RC2/w_2020_50/DM-75643`` or ``DECam/runs/HiTS-2015/d_2021_90/DM-80000``.
+These versions are intended to make it easy for users to browse collections and understand what is in them at a glance; formal provenance for software versions actually used in the processing will be automatically stored in the data repository itself.
+Of course, the version in the collection name should differ as little as possible from the versions actually used to reduce confusion.
+
+These names should always correspond to a "public" ``CHAINED`` collection that aggregates both all ``RUN`` collections that directly hold outputs and all collections used as inputs.
+The organization of those "private" output ``RUN`` collections (if there is more than one) is completely at operator discretion, though these collections should start with the same prefix as the umbrella ``CHAINED`` collection, followed by a slash.
+
+In cases where one or more private ``RUN`` collections contain datasets that should not be considered part of the final public outputs (e.g. because they are superceded by datasets in other private ``RUN`` collections), a ``TAGGED`` collection can be used to screen and aggregate these.
+That ``TAGGED`` collection would then be a direct child of the final public ``CHAINED`` collection, instead of any ``RUN`` collections it references.
+
+.. note::
+
+   It is not generally possible to use a ``TAGGED`` collection as the public output collection for a processing run, because putting master calibrations (which are almost always inputs, even if indirectly) in a ``TAGGED`` collection strips them of their validity ranges and does not allow datasets from different validity ranges to coexist.
+   So even if a ``TAGGED`` collection is used, the public ``CHAINED`` collection would contain both that collection and the input ``CALIBRATION`` collection as children.
```

Comment:
It's actually fundamentally impossible to write a dataset to a `TAGGED` collection, or any kind of collection other than a `RUN`.  They aren't really like git tags (the name comes from DESDM nomenclature for the kind of database table in which they are implemented, though it may well extend well beyond DESDM).  But they are like a set of file symlinks in the same sense that a `RUN` is like a directory, so I'd say that "writing to a TAGGED collection" is invalid in the same sense that writing a file "to a set of symlinks" is invalid.